### PR TITLE
Feature/merge 0.7.1 master

### PR DESCRIPTION
**Pull request by improbable-danny**
**Contributions**: We are not currently taking public contributions - see our [contributions](CONTRIBUTING.md) policy. However, we are accepting issues and we do want your [feedback](../README.md#give-us-feedback).

-------

#### Description
Merging gRPC port changes and China support to master 

#### Tests
Cloud deployment 

STRONGLY SUGGESTED: How can this be verified by QA?

#### Primary reviewers
@oblm

### CODE REVIEW

File: SpatialGDK/Source/SpatialGDKServices/Private/LocalDeploymentManager.cpp [hunk 1]
```diff
@@ -23,7 +23,21 @@ DEFINE_LOG_CATEGORY(LogSpatialDeploymentManager);
 
 #define LOCTEXT_NAMESPACE "FLocalDeploymentManager"
 
-static const FString SpatialServiceVersion(TEXT("20190930.180414.3b04a59226"));
+static const FString SpatialExe(TEXT("spatial.exe"));
```
Comment (joshuahuburn) [inline SpatialGDK/Source/SpatialGDKServices/Private/LocalDeploymentManager.cpp]: Why is this here? Methods should be using `FSpatialGDKServicesModule::GetSpatialExe()` in 0.7

Comment (improbable-danny) [reply SpatialGDK/Source/SpatialGDKServices/Private/LocalDeploymentManager.cpp]: Good spot, will remove

File: Setup.sh [hunk 2]
```diff
@@ -59,12 +61,12 @@ if [ -d "${SPATIAL_DIR}" ]; then
 fi
 
 echo "Retrieve dependencies"
-spatial package retrieve tools       schema_compiler-x86_64-macos        "${PINNED_CORE_SDK_VERSION}"  "${CORE_SDK_DIR}"/tools/schema_compiler-x86_64-macos.zip
-spatial package retrieve schema      standard_library                    "${PINNED_CORE_SDK_VERSION}"  "${CORE_SDK_DIR}"/schema/standard_library.zip
-spatial package retrieve worker_sdk  c_headers                           "${PINNED_CORE_SDK_VERSION}"  "${CORE_SDK_DIR}"/worker_sdk/c_headers.zip
-spatial package retrieve worker_sdk  c-dynamic-x86_64-clang-macos        "${PINNED_CORE_SDK_VERSION}"  "${CORE_SDK_DIR}"/worker_sdk/c-dynamic-x86_64-clang-macos.zip
-spatial package retrieve worker_sdk  c-static-fullylinked-arm-clang-ios  "${PINNED_CORE_SDK_VERSION}"  "${CORE_SDK_DIR}"/worker_sdk/c-static-fullylinked-arm-clang-ios.zip
-spatial package retrieve worker_sdk  csharp                              "${PINNED_CORE_SDK_VERSION}"  "${CORE_SDK_DIR}"/worker_sdk/csharp.zip
+spatial package retrieve tools       schema_compiler-x86_64-macos        "${PINNED_CORE_SDK_VERSION}"  ${DOMAIN_ENVIRONMENT_VAR}  "${CORE_SDK_DIR}"/tools/schema_compiler-x86_64-macos.zip
```
Comment (improbable-danny) [inline Setup.sh]: @jessicafalk I haven't been able to test this, does this change look alright to you?

File: SpatialGDK/Source/SpatialGDKEditor/Public/SpatialGDKEditorSettings.h [hunk 3]
```diff
@@ -465,4 +479,6 @@ class SPATIALGDKEDITOR_API USpatialGDKEditorSettings : public UObject
 	}
 
 	bool IsDeploymentConfigurationValid() const;
+
+	FORCEINLINE bool IsRunningInChina() const { return ServicesRegion == EServicesRegion::CN; }
```
Comment (joshuahuburn) [inline SpatialGDK/Source/SpatialGDKEditor/Public/SpatialGDKEditorSettings.h]: FYI @improbable-danny. @jessicafalk has merged a `SpatialGDKServicesSettings` file in her PR which will allow us to refactor this in the future. Could you spin up a ticket :D

File: CHANGELOG.md [hunk 4]
```diff
@@ -44,6 +44,17 @@ Usage: `DeploymentLauncher createsim <project-name> <assembly-name> <target-depl
 - Unresolved object references in replicated arrays of structs should now be properly handled and eventually resolved.
 - Fix tombstone-related assert that could fire and bring down the editor.
 
+## [`0.7.1`] - 2019-12-05
+
+### Bug fixes: 
+- The C++ Worker SDK now communicates on port 443 instead of 444. This change is intended to protect your cloud deployments from DDoS attacks.
```
Comment (oblm) [inline CHANGELOG.md]:
```suggestion
- The C Worker SDK now communicates on port 443 instead of 444. This change is intended to protect your cloud deployments from DDoS attacks.
```

File: CHANGELOG.md [hunk 5]
```diff
@@ -44,6 +44,17 @@ Usage: `DeploymentLauncher createsim <project-name> <assembly-name> <target-depl
 - Unresolved object references in replicated arrays of structs should now be properly handled and eventually resolved.
 - Fix tombstone-related assert that could fire and bring down the editor.
 
+## [`0.7.1`] - 2019-12-05
```
Comment (oblm) [inline CHANGELOG.md]:
```suggestion
## [`0.7.1-preview`] - 2019-12-00
```

**Comment by joshuahuburn** [comment]
Closing this and will remake after preview has been merged.
